### ISSUE TITLE
South Norfolk and Broadland UK no longer working

### ISSUE THREAD

**Issue by ThomasPrior** [mampfes/hacs_waste_collection_schedule#1103]
Since updating to 1.41.0 I am unable to fetch waste collection data from South Norfolk and Broadland District Council.

Last working version: 1.40.0
Installation method: HACS
Workaround: revert to 1.40.0 and the previously used address_payload configuration.

Error:

```2023-07-19 14:11:25.390 ERROR (SyncWorker_1) [waste_collection_schedule.source_shell] fetch failed for source Broadland District Council:
  File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source_shell.py", line 134, in fetch
  File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source/south_norfolk_and_broadland_gov_uk.py", line 89, in fetch
  File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source/south_norfolk_and_broadland_gov_uk.py", line 93, in __fetch_by_postcode_and_address
```

Testing using the test config fails in the same way as my actual address.

configuration.yaml:

```
waste_collection_schedule: !include waste_collection.yaml
```

waste_collection.yaml:

```
sources:
  - name: south_norfolk_and_broadland_gov_uk
    args:
      address: 14 Fairland Street, Wymondham, Norfolk, NR18 0AW
      postcode: NR18 0AW
```

**Comment by 5ila5** [comment]
fixed by #1104. Not sure how this slipped through.
Available now as master/git version

**Comment by ThomasPrior** [comment]
Very much appreciated, thank you!
